blocks/cards/cards.css

```diff
@@ -246,10 +247,19 @@ main .cards-wrapper .machine-parts .cards-card-body h3{
     padding-right: 0;
   }
 
+  .cards > ul {  
+    flex-flow: row wrap;
+  }
 
   .cards > ul > li {
     padding: 0;
     text-align: center;
+    flex: 0 0 calc(100% / 3 - 2rem);
```

Comment:
This assumes we'd have 3 cards in the block and would not work if the number of cards is not 3. How about we use `width` property here instead?